src/components/AmountFilter.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { Button, Popover, InputLabel, Grid } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import RemoveIcon from '@material-ui/icons/Remove';
+import AmountTextField from './AmountTextField';
+
+const useStyles = makeStyles((theme) => ({
+  buttonStyle: {
+    margin: theme.spacing(1),
+    maxWidth: 120,
+    height: 57,
```

Comment:
What is this?

Reply:
It sets the height of the Filter button. It looks weird without it. 
<img width="443" alt="Screen Shot 2021-01-23 at 11 08 45 PM" src="https://user-images.githubusercontent.com/40483569/105623619-f67e0700-5dcf-11eb-9428-03630277f4ef.png">

Reply:
Hmm, 57 is a rather magical number (also from the inspector it looks like the actual size is _56_) but it looks like this is due to the other selects having the [outlined](https://material-ui.com/components/selects/#simple-select) variant as the non-variant ones have a height of 32px.

Instead of introducing a magical number I suggest removing the `variant="outlined"` attribute from the other elements so that there isn't such a height discrepancy. You may need to additionally vertically center the elements on the left as it looks like [buttons](https://material-ui.com/components/buttons/) are usually 36px (you can look at how it's done on the right side of the FilterBar for reference).